cms/templates/partials/catalog-card.html

```diff
@@ -0,0 +1,93 @@
+{% load static humanize wagtailcore_tags wagtailimages_tags %}
+<div class="catalog-card" role="region" aria-label="{{ courseware_page.title }}">
+    <div class="top">
+        <div class="cover-image">
+            <div class="cover-image-frame">
+                {% if courseware_page.thumbnail_image %}
+                    {% image courseware_page.thumbnail_image fill-335x203 %}
+                {% else %}
+                    <img src="{% static default_image_path %}" alt="Preview image">
+                {% endif %}
+            </div>
+        </div>
+        <div class="top-level-details">
+            <h3 class="title">
+                    <a href="{% pageurl courseware_page %}">
+                        {{ courseware_page.title }}
+                    </a>
+            </h3>
+            <ul>
+                {% if courseware_page.product.current_price %}
+                    <li>
+                        <strong>Cost:</strong>
+                        ${{ courseware_page.product.current_price|floatformat:"0"|intcomma }}
+                    </li>
+                {% endif %}
+                {% if object_type == "program" %}
+                    <li>
+                        <strong>Number of Courses:</strong>
+                        {{ courseware_page.product.courses.count }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.duration %}
+                    <li>
+                        <strong>Duration:</strong>
+                        {{ courseware_page.duration }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.product.next_run_date %}
+                    <li>
+                        <strong>Next Start Date:</strong>
+                        {{ courseware_page.product.next_run_date|date:"F j, Y" }}
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+        <div class="offering-type">
+            <label>
+                {% if object_type == "course" %}
+                    Single
+                {% else %}
+                    Multi
+                {% endif %}
+                Course Online
+            </label>
+        </div>
+    </div>
+    <div class="bottom">
+      {% with page_id=courseware_page.pk|stringformat:"i" %}
+        {% with collapse_id="collapsible_"|add:tab|add:"_"|add:page_id|add:page_id collapse_input_id="collapse_"|add:tab|add:page_id %}
```

Comment:
1. Looks like you're adding `page_id` twice to the first var: `add:page_id|add:page_id`
2. Any reason we can't combine these definitions with the `page_id` definition above to make one `with` block instead of 2?

Reply:
1. That' a typo, will fix it in the next commit
2. Will update.
3. The `tab` variable is string and `courseware_page.pk` is an int. Using both in the same chain of `add` filter does not play nice with string concatenation so we have to convert it to a string before we can use it.

Reply:
Updated.